assets/js/util.js

```diff
@@ -162,6 +164,11 @@ $(document).ready(function () {
         return false;
     });
 
+    $('#originalText').mouseup(function () {
+        if($('#originalText').height() < originalTextScrollHeight) {
+            $('#originalText').css('overflow-y', 'scroll');
```

Comment:
Why can't this situation be solved with simply a `overflow: y-scroll`? I'm guessing you've tried it.